### ISSUE TITLE
Suggest a feature!

### ISSUE THREAD

**Issue by drex44** [drex44/good-food-guide#7]
create or edit a file named feature_requests.md if you have any suggestion for this repo or website.

Read the readme.md for more information or comment if you need more information or help!

**Comment by plug-n-play** [comment]
i think creating Github issues for same is a better option unless the intention is just to count the PRs for "hacktoberfest" :)

**Comment by drex44** [comment]
@plug-n-play you're right to have suggestions as Github issues but I wanted to promote hacktoberfest so put together this issue. 
I thought people who submitted good features also should be rewarded. so submitting a PR will increase their PR count. while on the other hand, anybody with the right skills can implement it and submit a PR. so it will be a win-win situation for both contributors(Who gave the idea and who implemented it).

@plug-n-play, What are your thoughts on this?
if there are too many feature_requests coming then I will close this issue for some time till we catch up.
